### DISCUSSION TITLE
Using AutoExpert in OpenAI Assistants

### DISCUSSION THREAD

**Discussion by robertfalconer** [spdustin/ChatGPT-AutoExpert#100] [Q&A]
How would you best make use of these instructions to create an Assistant using the new OpenAPI Assistant API? As a first step, I tried pasting both stages of the instructions into the single instructions box, but the result was not formatted well (no table, newlines, etc) and the contents of the response wasn't anywhere near as comprehensive as the examples I have seen.

**Comment by spdustin** [comment]
I have a different configuration for the Assistants API that I've been picking at. Would you be interested in trying it out when I am closer to finished?
